react-design/my-app/src/Carousel.jsx

```diff
@@ -0,0 +1,11 @@
+export default function Carousel() {
```

Comment:
To make Phase 2 easier for you: 

Via Phase 1 tips: 

"The Indicators (in your case, Carousel) component renders a list of buttons, each displaying an _**index**,_ for cycling to a specific item; the current item will be highlighted"

I would refactor to pass a prop from the parent component of `count = {items.length} `

This way, we can dynamically create buttons based on how many items we have!